src/VirtoCommerce.Platform.Web/Telemetry/UserTelemetryInitializer.cs

```diff
@@ -10,9 +12,12 @@ namespace VirtoCommerce.Platform.Web.Telemetry
     /// </summary>
     public class UserTelemetryInitializer : TelemetryInitializerBase
     {
-        public UserTelemetryInitializer(IHttpContextAccessor httpContextAccessor)
+        private readonly IConfiguration _configuration;
+
+        public UserTelemetryInitializer(IHttpContextAccessor httpContextAccessor, IConfiguration configuration)
```

Comment:
inject `IOptions<ApplicationInsightsOptions>` instead of IConfiguration
add field ApplicationInsightsOptions _options instead instead of IConfiguration